Sources/PointFree/Episode/Show.swift

```diff
@@ -628,75 +622,55 @@ private let transcriptView = View<([Episode.TranscriptBlock], Bool)> { blocks, i
   )
 }
 
-private func transcript(blocks: [Episode.TranscriptBlock], isEpisodeViewable: Bool) -> [Node] {
+private func transcript(blocks: [Episode.TranscriptBlock], isEpisodeViewable: Bool) -> Node {
   struct State { var nodes: [Node] = [], titleCount = 0 }
 
-  return blocks
-    .reduce(into: State()) { state, block in
-      if case .title = block.type { state.titleCount += 1 }
-      state.nodes += state.titleCount <= 1 || isEpisodeViewable
-        ? transcriptBlockView.view(block)
-        : []
-    }
-    .nodes + subscriberCalloutView(isEpisodeViewable: isEpisodeViewable)
+  return [
+    .fragment(
+      blocks
+        .reduce(into: State()) { state, block in
+          if case .title = block.type { state.titleCount += 1 }
+          state.nodes += state.titleCount <= 1 || isEpisodeViewable
+            ? [transcriptBlockView.view(block)]
+            : []
+        }
+        .nodes
+    ),
+    subscriberCalloutView(isEpisodeViewable: isEpisodeViewable)
+  ]
 }
 
-private func subscriberCalloutView(isEpisodeViewable: Bool) -> [Node] {
+private func subscriberCalloutView(isEpisodeViewable: Bool) -> Node {
   guard !isEpisodeViewable else { return [] }
 
-  return [
-    gridRow([
-      gridColumn(
-        sizes: [.mobile: 12],
-        [style(margin(leftRight: .auto))],
-        [
-          div(
-            [
-              `class`(
-                [
-                  Class.margin([.mobile: [.top: 4]]),
-                  Class.padding([.mobile: [.all: 3]]),
-                  Class.pf.colors.bg.gray900
-                ]
-              )
-            ],
-            [
-              h4(
-                [
-                  `class`(
-                    [
-                      Class.pf.type.responsiveTitle4,
-                      Class.padding([.mobile: [.bottom: 2]])
-                    ]
-                  )
-                ],
-                ["Subscribe to Point-Free"]
-              ),
-              p(
-                [
-                  "👋 Hey there! Does this episode sound interesting? Well, then you may want to ",
-                  a(
-                    [
-                      href(path(to: .pricing(nil, expand: nil))),
-                      `class`([Class.pf.type.underlineLink])
-                    ],
-                    ["subscribe"]
-                  ),
-                  " so that you get access to this episodes and more!",
-                  ]
-              )
-            ]
-          )
-        ]
+  return gridRow(
+    gridColumn(
+      sizes: [.mobile: 12],
+      [style(margin(leftRight: .auto))] as [Attribute<Tag.Div>],
```

Comment:
Not sure why this annotation is required. But the compiler just errors for me otherwise.